packages/design-system/src/connect/components/document-toolbar/document-toolbar.tsx

```diff
@@ -1,56 +1,59 @@
 import { Icon } from "@powerhousedao/design-system";
+import {
+  exportDocument,
+  setSelectedNode,
+  setSelectedTimelineItem,
+  showRevisionHistory,
+  useDocumentTimeline,
+  useNodeParentFolderById,
+  useSelectedDocument,
+} from "@powerhousedao/reactor-browser";
+import type { PHDocument } from "document-model";
 import { useEffect, useState } from "react";
 import { twMerge } from "tailwind-merge";
-import type {
-  DocumentTimelineProps,
-  TimelineBarItem,
-  TimelineDividerItem,
-} from "../document-timeline/document-timeline.js";
 import { DocumentTimeline } from "../document-timeline/document-timeline.js";
+import { useDocumentUndoRedo } from "./utils/use-document-undo-redo.js";
 
 export type DocumentToolbarProps = {
-  title?: string;
+  document?: PHDocument;
   className?: string;
-  undo?: () => void;
-  redo?: () => void;
-  canUndo?: boolean;
-  canRedo?: boolean;
-  onExport?: () => void;
-  onClose: () => void;
-  onShowRevisionHistory?: () => void;
-  timelineItems?: Array<TimelineBarItem | TimelineDividerItem>;
+  disableRevisionHistory?: boolean;
   onSwitchboardLinkClick?: () => void;
   initialTimelineVisible?: boolean;
   timelineButtonVisible?: boolean;
-  onTimelineItemClick?: DocumentTimelineProps["onItemClick"];
+  onClose?: () => void;
 };
 
 export const DocumentToolbar: React.FC<DocumentToolbarProps> = (props) => {
```

Comment:
Please add some comments here and on each prop so the dev can read about the component when hovering it.